inst/include/dust/r/dust.hpp

```diff
@@ -470,7 +470,7 @@ cpp11::writable::doubles run_filter(T * obj,
 template <typename T, typename std::enable_if<!std::is_same<dust::no_data, typename T::data_type>::value, int>::type = 0>
 cpp11::sexp dust_filter(SEXP ptr, bool save_trajectories,
                         cpp11::sexp r_step_snapshot) {
-  // typedef typename T::real_type real_type;
+  // using real_type = typename T::real_type;
```

Comment:
Do we need this line/comment?

Reply:
Almost certainly not, will get rid of that